### ISSUE TITLE
[148.5] Implement Yoda's JukeboxCreatedSubscriber

### ISSUE THREAD

**Issue by peddybeats** [PedramPejman/jukebox#153]
- This is the follow-up issue for the [design issue](https://github.com/PedramPejman/jukebox/issues/150)
- @PeymanPejman after you have fleshed out the design, comment the high-level structure here and we'll work out the kinks, if any.
- Note that you will not be able to test your implementation until issue 148.1-4 are done.
- Once this is done, go ahead and close 148 :)

**Comment by peddybeats** [comment]
#157
